Misc/NEWS.d/next/Tests/2023-10-01-10-27-02.gh-issue-110171.ZPlo0h.rst

```diff
@@ -0,0 +1 @@
+``libregrtest`` now always sets ``random.seed``.
```

Comment:
Since this message is for end users, would you mind to elaborate it? I suppose that the impact for *end users* is that test failure sare now easier to reproduce using the random seed. Previously, you may not be able to reproduce it, since the seed was unknown, right?

Reply:
Done! 👍